plugins/mcreator-core/procedures/entity_get_stepheight.json

```diff
@@ -0,0 +1,27 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "entity",
+      "check": "Entity"
+    }
+  ],
+  "inputsInline": true,
+  "output": "Number",
+  "colour": "%{BKY_MATH_HUE}",
+  "mcreator": {
+    "toolbox_id": "entitydata",
+    "toolbox_init": [
+      "<value name=\"entity\"><block type=\"entity_from_deps\"></block></value>"
+    ],
+    "inputs": [
+      "entity"
+    ],
+    "dependencies": [
```

Comment:
Input provides dependency, no need for procedure block to do this, remove this as this will break dependency system with ghost dependencies

Reply:
![image](https://user-images.githubusercontent.com/16374228/222246278-318c1fdf-3a3d-4cc7-bf7a-fbb810ff3a26.png)

No need for entity here, yet procedure declares it